data_juicer/utils/model_utils.py

```diff
@@ -255,13 +255,36 @@ def decompress_model(compressed_model_path):
     return diversity_model
 
 
+def prepare_diffusion_model(model_name_or_path):
```

Comment:
Recommended to align with the official interface of `diffusers`, using `pretrained_model_name_or_path`.